src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -20,14 +20,36 @@ internal class LiveLogger : ILogger
         public LoggerVerbosity Verbosity { get; set; }
         public string Parameters { get; set; }
 
+        /// <summary>
+        /// List of events the logger needs as parameters to the <see cref="ConfigurableForwardingLogger"/>.
+        /// </summary>
+        /// <remarks>
+        /// If LiveLogger runs as a distributed logger, MSBuild out-of-proc nodes might filter the events that will go to the main node using an instance of <see cref="ConfigurableForwardingLogger"/> with the following parameters.
+        /// </remarks>
+        public static readonly string[] ConfigurableForwardingLoggerParameters =
```

Comment:
I think it might be helpful to group these by started/finished, so  `BUILDSTARTEDEVENT;BUILDFINISHEDEVENT` could be next to each other.